lib/rmg/CRmgTemplate.h

```diff
@@ -116,7 +116,7 @@ class DLL_LINKAGE ZoneOptions
 
 	ZoneOptions();
 
-	ZoneOptions & operator=(const ZoneOptions & other);
+	ZoneOptions & operator=(const ZoneOptions & other) = default;
```

Comment:
Can be removed entirely?

Reply:
@IvanSavenko why? We do not need to assign ZoneOptions?

Reply:
Unless I am missing something, assignment operator is one of compiler-generated methods?

Reply:
In this cause you are right, but compiler generates assignments in much less places than constructors.

Reply:
removed.